app/controllers/master_files_controller.rb

```diff
@@ -206,6 +202,24 @@ def attach_captions
     end
   end
 
+  def delete_captions
+    if params[:id].blank? || (not MasterFile.exists?(params[:id]))
+      flash[:notice] = "MasterFile #{params[:id]} does not exist"
+    end
+    @master_file = MasterFile.find(params[:id])
```

Comment:
I know you didn't start this pattern but many methods in this controller use this same code block. I think we should consolidate them into a private/protected method and use a `before_action` to hook it up.

Reply:
Yeah sounds good :100: